src/monorepo_manifest.rs

```diff
@@ -47,34 +51,37 @@ where
     // ignore paths to speed up file-system walk
     package_manifests.push(String::from("!node_modules/"));
 
+    let monorepo_root = directory.as_ref().to_owned();
+
     GlobWalkerBuilder::from_patterns(&directory, &package_manifests)
         .file_type(FileType::FILE)
         .min_depth(1)
         .build()
         .expect("Unable to create glob")
         .into_iter()
         .filter_map(Result::ok)
-        .map(|dir_entry| {
+        .parallel_map(move |dir_entry| {
```

Comment:
Since this is IO-bound, it probably will benefit from running on more than just number of CPU cores threads. I'd try 32. Use https://docs.rs/pariter/latest/pariter/trait.IteratorExt.html#method.parallel_map_custom

Reply:
https://docs.rs/pariter/latest/pariter/index.html#customizing-settings

Reply:
I had noticed through trial and error that 32 threads provided better performance, but I didn't think about why that would be the case. Makes sense, thanks for the feedback!